pages/api/shares.py

```
@@ -0,0 +1,111 @@
+from lib.database import execute_query
+from upload_album import app  # Import app từ file upload_album.py
+
+@app.route('/api/shares', methods=['POST'])
+def create_share():
+    try:
+        data = request.get_json()
+        album_id = data.get('albumId')
+        username = data.get('username')
+        
+        # Debug log
+        print(f"\n=== Debug create_share ===")
```

Ruff (0.7.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)